.github/workflows/on_PR_meson_clang.yaml

```diff
@@ -9,12 +9,11 @@ concurrency:
 jobs:
   Ubuntu:
     runs-on: ubuntu-20.04
-    name: Linux-Clang${{matrix.cxx}}-${{matrix.deps}}-${{matrix.type}}
+    name: Linux-Clang${{matrix.cxx}}-${{matrix.deps}}
     strategy:
       matrix:
         cxx: ['7', '8', '9', '10', '12']
```

Comment:
Let's bump this to ubuntu-22.04 and some newer LLVM versions?

Reply:
And same for the GCC matrix as well, I don't see the point in checking/supporting this far back...

Reply:
If it's possible without too much headache, why not?

Newer compilers are tested with other CI.

Reply:
I'd still prefer to use the latest LTS whenever possible (i.e. both of these workflows). Older compiler are also tested w/ other CI.

Reply:
Beyond the scope of this PR.